ts/src/index.ts

```diff
@@ -148,13 +151,27 @@ export async function startLocal(config: Config = {}): Promise<void> {
     if (!config.disableHeap) {
       const heap = await profiler.profile(
           {name: 'HEAP-Profile' + new Date(), profileType: 'HEAP'});
+      const decodedBytes = Buffer.from(heap.profileBytes as 'string', 'base64');
+      fs.writeFile(
```

Comment:
Would be good to document somewhere in the code where the profiles will get written (current directory? temp directory?)

Reply:
Done